File: app/android/app/src/main/java/com/jun/weather/repository/web/api/entity/ForecastItem.java
package com.jun.weather.repository.web.api.entity;

import com.google.gson.annotations.SerializedName;

public class ForecastItem extends ResponseItem{
    @SerializedName("fcstDate")
    public String fcstDate;

    @SerializedName("fcstTime")
    public String fcstTime;

    @SerializedName("fcstValue")
    public String fcstValue;
}

File: app/android/app/src/main/java/com/jun/weather/repository/web/entity/ForecastItem.kt
package com.jun.weather.repository.web.entity

data class ForecastItem(
        val baseDate: String,
        val baseTime: String,
        val category: String,
        val nx: Int,
        val ny: Int,
        val fcstDate: String,
        val fcstTime: String,
        val fcstValue: String
)